docs/tables/aws_elb_access_log/index.md

```diff
@@ -0,0 +1,180 @@
+---
+title: "Tailpipe Table: aws_elb_access_log - Query AWS ELB Access Logs"
+description: "AWS ELB Access logs capture detailed information about the requests that are processed by an Elastic Load Balancer. This table provides a structured representation of the log data, including request and response details, client and target information, processing times, and security parameters."
+---
+
+# Table: aws_elb_access_log - Query AWS ELB access logs
+
+The `aws_elb_access_log` table allows you to query AWS Elastic Load Balancer (ELB) access logs. This table provides detailed information about requests processed by your load balancers, including client and target details, processing times, and security parameters.
+
+## Configure
+
+Create a [partition](https://tailpipe.io/docs/manage/partition) for `aws_elb_access_log` ([examples](https://hub.tailpipe.io/plugins/turbot/aws/tables/aws_elb_access_log#example-configurations)):
+
+```sh
+vi ~/.tailpipe/config/aws.tpc
+```
+
+```hcl
+connection "aws" "logging_account" {
+  profile = "my-logging-account"
+}
+
+partition "aws_elb_access_log" "my_elb_logs" {
+  source "aws_s3_bucket" {
+    connection = connection.aws.logging_account
+    bucket     = "aws-elb-logs-bucket"
+  }
+}
+```
+
+## Collect
+
+[Collect](https://tailpipe.io/docs/manage/collection) logs for all `aws_elb_access_log` partitions:
+
+```sh
+tailpipe collect aws_elb_access_log
+```
+
+Or for a single partition:
+
+```sh
+tailpipe collect aws_elb_access_log.my_elb_logs
+```
+
+## Query
+
+**[Explore example queries for this table →](https://hub.tailpipe.io/plugins/turbot/aws/queries/aws_elb_access_log)**
+
+### Failed requests
+
+Find failed HTTP requests (with status codes 400 and above) to troubleshoot load balancer issues.
+
+```sql
+select
+  timestamp,
+  elb,
+  client_ip,
+  target_ip,
+  elb_status_code,
+  target_status_code,
+  request
+from
+  aws_elb_access_log
+where
+  elb_status_code >= 400
+order by
+  timestamp desc;
+```
+
+### Slow response times
+
+Identify requests with high processing times that might indicate performance issues.
```

Comment:
What do we consider "high processing times"? It's not clear to me from the `where` clause

Reply:
In the WHERE clause, we calculate the total processing time by summing the request processing time, target response time, and response processing time:
	•	TargetResponseTime: Time taken by the target to process the request.
	•	RequestProcessingTime: Time taken by the load balancer to process the request.
	•	ResponseProcessingTime: Time taken to process the response.
The query filters requests where the combined processing time (request + target + response) exceeds 1 second. Will update the query description and add proper query comments for clarity.